subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/versionupgrade/CreatePullRequest.java

```diff
@@ -2,11 +2,14 @@
 
 import org.gradle.api.logging.Logger;
 import org.gradle.api.logging.Logging;
+import org.gradle.internal.impldep.org.apache.commons.lang.StringUtils;
```

Comment:
There is StringUtil class internally in Shipkit so no need to make a dependency on Gradle internals.